test/unit/api_docs/service_test.rb

```diff
@@ -547,6 +547,13 @@ class ApiDocs::ServiceTest < ActiveSupport::TestCase
     assert_same_elements api_docs[0..1].map(&:id), ApiDocs::Service.accessible.pluck(:id)
   end
 
+  test 'scope without_service' do
+    api_docs_services = FactoryGirl.create_list(:api_docs_service, 2)
```

Comment:
This will create 2 provider accounts no?

Reply:
2 `:simple_provider`, yes 😄